src/main/java/org/jenkinsci/plugins/workflow/support/steps/FilePathDynamicContext.java

```diff
@@ -45,7 +45,7 @@
  * Allows a step body to save a representation of a workspace
  * without forcing a particular {@link FilePath#getChannel} to be used the whole time.
  */
-@Extension public final class FilePathDynamicContext extends DynamicContext.Typed<FilePath> {
+@Extension(ordinal = 100) public final class FilePathDynamicContext extends DynamicContext.Typed<FilePath> {
```

Comment:
Is this needed for `PushdStep` to keep working, or something else?

Reply:
Yes exactly that.